CHANGES_NEXT_RELEASE

```diff
@@ -14,3 +14,4 @@
 - [cygnus][doc] Replace references to JDK 1.6 with JDK 1.7 (#1232)
 - [cygnus][hardening] Add NGSIHDFSSink configuration in docker's agent.conf file (#1172)
 - [cygnus][hardening] Fix NGSIMySQLSink configuration in docker's agent.conf file (#1237)
+- [cygnus][hardening] Fix use of Apache Flume httpclient,httpcore and libthrift old libs (#1233)
```

Comment:
I think it is more descriptive: "Force the usage of certain version in libraries used by cygnus-ngsi docker"